<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HardwareInfoDialog</class>
 <widget class="QDialog" name="HardwareInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HardwareInfoDialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Graphics Card:</string>
   </property>
  </widget>
  <widget class="QLabel" name="usedMemoryLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>used</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>142</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Used memory:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="usedMemoryProgressBar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>175</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>202</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGraphics">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="totalMemoryLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>total</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>HardwareInfoDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
